docs/pages/versions/unversioned/sdk/application.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Application
+---
+
+This module provides useful information about the native application, such the bundle ID, app name, and build version.
+
+## Installation
+
+This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. To use it in a [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native app, follow its [installation instructions](https://github.com/expo/expo/tree/master/packages/expo-device).
+
+## API
+
+```js
+import * as Application from 'expo-application';
+```
+
+### Constants
+
+- [`Application.applicationName`](#applicationapplicationname)
+- [`Application.bundleId`](#applicationbundleid)
+- [`Application.nativeAppVersion`](#applicationnativeappversion)
+- [`Application.nativeBuildVersion`](#applicationnativebuildversion)
+- [`Application.androidId`](#applicationandroidid) (Android only)
+
+### Methods
+
+- [`Application.getIosIdForVendorAsync()`](#applicationgetiosidforvendorasync) (iOS only)
+- [`Application.getInstallReferrerAsync()`](#applicationgetinstallreferrerasync) (Android only)
+- [`Application.getFirstInstallTimeAsync()`](#applicationgetfirstinstalltimeasync)
+- [`Application.getLastUpdateTimeAsync()`](#applicationgetlastupdatetimeasync) (Android only)
+
+## Constants
+
+### `Application.applicationName`
+
+The human-readable name of the application.
```

Comment:
Need to be comprehensive -- the easy stuff is easy to guess anyway so most of the value in the docs is the stuff that's hard to find out.

```suggestion
The human-readable name of the application that is displayed with the app's icon on the device's home screen or desktop. On Android and iOS, this value is a string unless the name could not be retrieved, in which case this value will be null. On web this value is null.
```